projects/observability/src/shared/dashboard/data/graphql/explore/explore-cartesian-data-source.model.ts

```diff
@@ -63,7 +63,7 @@ export abstract class ExploreCartesianDataSourceModel extends GraphQlDataSourceM
       requestType: EXPLORE_GQL_REQUEST,
       selections: requestState.series.map(series => series.specification),
       context: requestState.context,
-      limit: requestState.groupByLimit ?? 100,
+      limit: requestState.resultLimit,
```

Comment:
On the consumer side, do we need to change the existing limit in modelJsons (may be) since this will now limit the number of results?

Reply:
None exist in the UI as of today (besides what you've got in flight), but in general yes - new data models that extend this will have to use the two types of limit appropriately.